contracts/src/ethereum/L1Token.sol

```diff
@@ -4,7 +4,21 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 contract L1Token is ERC20 {
-    constructor(uint256 amount) ERC20("L1Token", "L1T") {
+
+    ERC20 public REWARD_TOKEN;
+
+    constructor(uint256 amount, address rewardToken) ERC20("L1Token", "L1T") {
         _mint(msg.sender, amount);
+        REWARD_TOKEN = RewAAVE(rewardToken);
```

Comment:
The contract `L1Token` represents any ERC20 that is accepted by Aave on L1 and L2, right? Those contracts won't have a reward token stored as storage variable. We could instead store it on L1 TokenBridge.

Reply:
I thought L1Token here represents the minimal version of StaticATokenLM that we need for testing.
We only need to represent static tokens since the bridge is intended only for static tokens.

Reply:
You're totally right, I got confused. It's indeed stored like that on `StaticATokenLM`.